app/_assets/entrypoints/application.js

```diff
@@ -39,5 +39,5 @@ document.addEventListener('DOMContentLoaded', (event) => {
     placement: 'left',
     icon: '#'
   });
-  anchors.add('.doc h1, .doc h2, .doc h3, .doc h4, .doc h5, .doc h6');
+  anchors.add('.doc h1:not(.no-anchor), .doc h2:not(.no-anchor), .doc h3:not(.no-anchor), .doc h4:not(.no-anchor), .doc h5:not(.no-anchor), .doc h6:not(.no-anchor)');
```

Comment:
why is this needed?

Reply:
it's to be able to mark some heading as one, which shouldn't be linked to, for example:
![image](https://github.com/user-attachments/assets/e065de88-53e4-482b-8480-2b0adfeaccd0)
it's in tabs so we can't link to it and if we allow `anchorjs` to process it, it would not do so correctly:
![image](https://github.com/user-attachments/assets/985534ea-bd67-4f30-957c-344a27f520d1)